server/lib/chat/chat-service.ts

```diff
@@ -477,7 +476,9 @@ export default class ChatService {
   ): Promise<SbUserId | null> {
     const { newOwnerId } = await removeUserFromChannel(userId, channelName)
 
-    // Notify the new owner that their permissions changed, but only if they're online.
+    // As a slight optimization, attempt to find the new owner in the database and notify them that
```

Comment:
So you've added a comment stating the why, but I still don't think this is worth doing, so that doesn't really address my concern. If we wanted to be totally maximally efficient on everything we could do little things like this and add comments everywhere, but the app would become extremely brittle for no real gain. The options here are:

- Utilize the data from a previous query (probably updating the query to return them in addition to what it returns now)
- Just make the new query always (it is *extremely* cheap)

With either of those options, you're always notifying the new owner and letting Nydus deal with no-oping the send if they're offline, which is the right division of responsibilities here.

Reply:
Oh, I already removed this unnecessary optimization, but it got left behind with my uncommitted changes for some reason. My bad ^^.